components/TaskContainer.vue

```diff
@@ -72,10 +71,12 @@ export default {
       if (this.taskDescription.length <= 0) {
         this.titleInputError = true;
         this.titleErrorMsg = 'Field is empty';
-      } else {
-        this.addTask();
+
+        return;
       }
 
+      this.addTask();
+
       e.preventDefault();
```

Comment:
this should be the first line?

Reply:
If block needs to be checked first to throw an error when the field is empty